dispatcher/frontend-ui/src/components/PipelineTable.vue

```diff
@@ -180,12 +180,20 @@
       },
       async loadLastRuns() {
         let parent = this;
+        parent.last_runs_loaded = false;
         let schedule_names = parent.tasks.map(function (item) { return item.schedule_name; }).unique();
 
         parent.toggleLoader("fetching last runs…");
-        let requests = schedule_names.map(function (schedule_name) {
-            return parent.queryAPI('get', "/schedules/" + schedule_name);
-          });
+        const chunkSize = 5;
+        let requests = []
+        for (let i = 0; i < schedule_names.length; i += chunkSize) {
+            const chunk = schedule_names.slice(i, i + chunkSize);
+            for (let iChunk in chunk) {
```

Comment:
```suggestion
            for (let schedule_name in chunk) {
```

Reply:
It is not yet a schedule name, it is only an index in the chunk

Reply:
I find it's clearer with the `iChunk` name

Reply:
My bad, I thought it was a schedule name